src/NTDll/NTDllExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <summary>
+    /// Extension methods available for and from the <see cref="NTDll"/> library.
+    /// </summary>
+    [OfferFriendlyOverloads]
+    public static partial class NTDllExtensions
```

Comment:
While the overload you're defining makes sense and feels consistent with the docs, we should not put it in a `*Extensions` class. Instead, we should put them in a partial class defined after the Helper filename pattern you can find elsewhere.

Reply:
Let's say I add this to `NTDll.Helpers.cs`:

```csharp
public static unsafe NTSTATUS RtlGetVersion(Kernel32.OSVERSIONINFOEX* versionInformation)
{
    return NTDll.RtlGetVersion((Kernel32.OSVERSIONINFO*)versionInformation);
}
```

That would generate an IntPtr overload:

```csharp
NTSTATUS RtlGetVersion(IntPtr versionInformation)
```

which clashes with the `IntPtr` overload generated for `NTSTATUS RtlGetVersion(Kernel32.OSVERSIONINFO* versionInformation)`

Thoughts? I can drop the the `OSVERSIONINFOEX` overload in this PR (I suggest to keep the struct, though) and raise a follow-up issue if that's easier.

Reply:
> I can drop the the OSVERSIONINFOEX overload in this PR (I suggest to keep the struct, though) and raise a follow-up issue if that's easier.

Ya, let's do that. I think this may require an enhancement to our code generator.